.travis.yml

```diff
@@ -96,16 +99,16 @@ matrix:
           - gfortran
     - python: 2.7
       env:
-        - TEST_AUTOWRAP="true"
-      sudo: true
-      virtualenv:
-        system_site_packages: true
-    - python: 2.7
-      env:
         - TEST_ASCII="true"
     - python: 3.4
       env:
         - TEST_ASCII="true"
+    - python: 2.7
+      env:
+        - TEST_AUTOWRAP="true"
```

Comment:
It's removing and installing python dependencies. It was moved to last to avoid it being uploaded as a cache. There's another option as well, `rm -f $CASHER_DIR/mtime.yml` (Source: https://developer.zendesk.com/blog/big-and-fast-tests-taking-our-travis-build-from-4-hours-to-13-minutes)